.pre-commit-config.yaml

```diff
@@ -4,7 +4,6 @@ repos:
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
-    -   id: check-yaml
```

Comment:
why?

Reply:
Cause it triggers on helmfile.yaml gotpl parts

Reply:
Hmm, it started works. Reverted that change